nl.json

```diff
@@ -1,192 +1,192 @@
+                "discord" : "Discord",
+                "twitter" : "Twitter"
+            },
+            "api" : "api",
+            "account" : {
+                "dropdown" : "account",
+                "account" : "account",
+                "logout" : "log uit"
+            },
+            "languagePicker" : {
+                "title" : "Kies een taal",
+                "currentLanguage" : "Je gebruikt nu [language]."
+            }
+        },
+        "home" : {
+            "title-upcoming" : "Items die eraan komen",
+            "title-all" : "Alle Items",
+            "error" : "We kunnen de items niet ophalen, probeer het later nog eens.",
+            "error-too-long" : "Dit duurt langer dan gedacht"
+        },
+        "shop" : {
+            "title" : "Huidige winkel",
+            "title-featured" : "Uitgelicht",
+            "title-daily" : "Dagelijks",
+            "error" : "Winkel niet beschikbaar, probeer het later opnieuw.",
+            "line1" : "De huidige item winkel van Fortnite Battle Royale - wordt dagelijks bijgewerkt om [00:00 UTC](https://time.is/UTC|tooltip=Huidige tijd in UTC|newtab=true).",
+            "line2" : "Je kan de item winkel van gisteren [hier](/shop/yesterday|tooltip=Item Winkel van gisteren) zien.",
+            "line3" : "Klik op een item voor meer informatie.",
+            "countdown" : {
+                "title" : "Nieuwe items:",
+                "wordyFormat" : "[number] [noun]",
+                "hour-singular" : "uur",
+                "hours-plural" : "uren"
+            }
+        },
+        "upcoming" : {
+            "title" : "Aankomende Items",
+            "line1" : "Alle items die nog niet zijn uitgekomen in Fortnite Battle Royale."
+        },
+        "list" : {
+            "title" : "Cosmetica Lijst",
+            "line1" : "Alle cosmetica beschikbaar in Fortnite Battle Royale.",
+            "table" : {
+                "icon" : "Icoon",
+                "name" : "Naam",
+                "type" : "Type",
+                "rarity" : "Zeldzaamheid",
+                "obtained" : "Verkregen"
+            }
+        },
+        "history" : {
+            "list" : {
+                "title" : "Item Winkel Geschiedenis",
+                "line1" : "Hier zijn de laatste 8 dagen van de item winkel voor Fortnite Battle Royale.",
+                "line2" : "Klik op een item om een hele winkel te zien of scroll om de volledige item winkel geschiedenis te zien per dag."
+            },
+            "view" : {
+                "title" : "Winkel Rotatie",
+                "line1" : "Dit is de item winkel rotatie van [date] voor Fortnite Battle Royale. ",
+                "line2" : "Klik op een item om meer informatie te zien.",
+                "today-button" : "Item Winkel van vandaag"
+            }
+        },
+        "news" : {
+            "title" : "Nieuws",
+            "line1" : "De huidige in-game nieuws feed van Epic Games. Dit wordt elke minuut geüpdate.",

```

Comment:
De correcte vorm is toch echt geüpdatet, zoek het maar op!

Reply:
> De correcte vorm is toch echt geüpdatet, zoek het maar op!

Oeps sorry, kan gebeuren!

Reply:
Maakt niets uit, dan weet je dat voor de volgende keer! We maken allemaal wel eens fouten.